cmd/xl-v1.go

```diff
@@ -155,6 +155,21 @@ func newXLObjects(storageDisks []StorageAPI) (ObjectLayer, error) {
 	xl.readQuorum = readQuorum
 	xl.writeQuorum = writeQuorum
 
+	// If the number of offline servers is equal to the readQuorum
+	// (i.e. the number of online servers also equals the
+	// readQuorum), we cannot perform quick-heal (no
+	// write-quorum). However reads may still be possible, so we
+	// skip quick-heal in this case, and continue.
+	offlineCount := 0
```

Comment:
perhaps you can use `len(newStorageDisks) - diskCount(newStorageDisks)`  or create a function similar to `diskCount` and rename it as `onlineDiskCount` and this one has `offlineDiskCount`